CONTRIBUTING.md

```diff
@@ -79,6 +79,8 @@ Here are some resources to help you get started on how to contribute code or new
 * ["Help wanted" issues](https://github.com/dotnet/aspnetcore/labels/help%20wanted) - these issues are up for grabs. Comment on an issue if you want to create a fix.
 * ["Good first issue" issues](https://github.com/dotnet/aspnetcore/labels/good%20first%20issue) - we think these are a good for newcomers.
 
+If you are a new member of the ASP.NET Core development team, use https://repos.opensource.microsoft.com/orgs/dotnet/teams/aspnet-push to get push access to the repository.
```

Comment:
Is the public documentation the right place to put this? Do we have any internal onboarding docs?

Reply:
As far as I'm aware, we don't have any internal docs. @captainsafia are you aware of any?

Reply:
Yeah, I don't think the public docs are the right place for this either.

That being said, we don't have any specific onboarding docs, although we could add it to our "Links for Engineers" wiki.

Reply:
👍🏻

Reply:
"Links for Engineers" - could you point me to where this is?

Reply:
https://github.com/dotnet/aspnetcore-internal/wiki/Links-for-Engineers

Reply:
Added here: https://github.com/dotnet/aspnetcore-internal/wiki/Links-for-Engineers#onboarding